.flake8

```diff
@@ -1,7 +1,6 @@
 [flake8]
 ignore =
   E501  # line too long, defer to black
-  F401  # unused import, defer to pylint
```

Comment:
Does this mean `pylint` wasn't checking for unused imports? Or was it just missing some?

Reply:
Pylint should [not have been looking for unused imports in `__init__.py`files](https://github.com/open-telemetry/opentelemetry-python-contrib/blob/main/.pylintrc#L241). I also found unused imports in non `__init__.py` files as well, so I don't know what is wrong with pylint.

Even so, it is better to check for unused imports with `flake8`, `pylint` is intrinsecally slow because it needs to install checked packages, `flake8` does not.